app/src/androidTest/java/com/willowtree/vocable/utility/PresetsViewModelTestDouble.kt

```diff
@@ -0,0 +1,45 @@
+package com.willowtree.vocable.utility
+
+import androidx.test.espresso.IdlingRegistry
+import com.willowtree.vocable.ICategoriesUseCase
+import com.willowtree.vocable.IPhrasesUseCase
+import com.willowtree.vocable.presets.Phrase
+import com.willowtree.vocable.presets.PresetsViewModel
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+import org.koin.core.context.loadKoinModules
+import org.koin.dsl.bind
+import org.koin.dsl.module
+
+class PresetsViewModelTestDouble(
+    categoriesUseCase: ICategoriesUseCase,
+    phrasesUseCase: IPhrasesUseCase,
+    private val idlingResource: MainActivityIdlingResource
+): PresetsViewModel(categoriesUseCase, phrasesUseCase) {
+    override suspend fun mapCategoryIdToPhrases(categoryId: String?): List<Phrase?> {
+        return super.mapCategoryIdToPhrases(categoryId).apply {
+            idlingResource.setIdle()
+        }
+    }
+}
```

Comment:
This is my only concern, as talked about IRL: this could be easily missed leading to test failures in the future, let's put this in-line with the ViewModel code.

Reply:
'Tis done.